akka-docs/src/main/paradox/typed/cluster-sharding.md

```diff
@@ -325,6 +325,29 @@ passivation strategy, and set the limit for active entities in a shard region:
 Or enable the least recently used passivation strategy and set the active entity limit using the
 `withLeastRecentlyUsedPassivationStrategy` method on `ClusterShardingSettings`.
 
+#### Segmented least recently used strategy
+
+A variation of the least recently used passivation strategy can be enabled that divides the active entity space into
+multiple segments to introduce frequency information into the strategy. Higher-level segments contain entities that
+have been accessed more often. The first segment is for entities that have only been accessed once, the second segment
+for entities that have been accessed at least twice, and so on. When an entity is accessed again, it will be promoted
+to the most recent position of the next-level or highest-level segment. The higher-level segments are limited, where
+the total limit is either evenly divided among segments, or proportions of the segments can be configured. When a
+higher-level segment exceeds its limit, the least recently used active entity tracked in that segment will be demoted
+to the level below. Only the least recently used entities in the lowest level will be candidates for passivation. The
+higher levels are considered "protected", where entities will have additional opportunities to be accessed before being
+considered for passivation.
```

Comment:
With this many strategies it's becoming difficult for the user to decide what to use. Can we add some short recommendation at the top of the `## Automatic Passivation` section. Maybe there is one that we think is "best" for most normal cases and we can recommend that to begin with? We probably want to highlight that we recommend changing to something else than the default idle strategy (it's only default for historical reasons).

Reply:
Yes. The next planned strategies will adapt automatically between recency-based and frequency-based, and should be good to use with most workloads. That can be the recommended default. For now, the SLRU strategy would make sense to use. I'm also thinking of reworking the configuration a little, so that there are named pre-configured strategies (like for other things), so we can have a `default-strategy` or similar that's already configured and can be switched to easily (and become the default for next akka minor version). Only the active entity limit would possibly need to be adjusted. In the docs, selecting other strategies or fine-tuning would become an additional more advanced topic.

I can rework things, and have the recommendation and easy switching to it, before extending to the advanced adaptive strategies.

Reply:
excellent, let's add that recommendation when the adaptive strategy is ready

Reply:
Reworking config to have a new default strategy, and setting up better for the advanced adaptive strategies, as a draft in #30968.